__tests__/app.test.js

```diff
@@ -295,38 +287,76 @@ test("400 - handles bad path - no such key", () => {
     .send({ inc_notes: 2 })
     .expect(400)
     .then(({ body: { msg } }) => {
-      expect(msg).toBe(
-        "Invalid - input must be in form {incl_votes: number}"
-      );
-    })
-  })
+      expect(msg).toBe("Invalid - input must be in form {incl_votes: number}");
```

Comment:
the correct format is {inc_votes: number} not incl_votes, this will mislead your user